### PR TITLE
Mark Security SIG as archived

### PR DESCRIPTION
**Pull request by fdegir**
This commit adds a new heading to the readme file, listing archived SIGs. Please note that SIG Security is listed as Archived in proposed.md even though the SIG is still shown as active in this readme file.

### CODE REVIEW

File: sigs/README.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ More details regarding SIG processes can be seen on [this link](https://github.c
...
 ## Current SIGS
 
-* [Security](https://github.com/cdfoundation/sig-security)
 * [MLOps](https://github.com/cdfoundation/sig-mlops)
 * [Interoperability](https://github.com/cdfoundation/sig-interoperability)
 * [Events](https://github.com/cdfoundation/sig-events)
+
+## Archived SIGs
+
+* [Security](https://github.com/cdfoundation/sig-security)
```
Comment (afrittoli) [inline sigs/README.md:22]: We can include the Chinese translation SIG here as well, but can also do it as a separate PR

**Review by afrittoli — APPROVED** [review]
Thank you @fdegir

File: sigs/README.md (hunk 1 above)
Comment (fdegir) [reply sigs/README.md:22]: Yep, it would perhaps be better to do that in a separate PR.
Updating the PR subject to state only SIG Security is marked as archived.
Also, moved sig-security.md into sigs/archived folder in the next commit.
